src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1021,16 +1021,8 @@ bool PolicyHandler::SendMessageToSDK(const BinaryMessage& pt_string,
   LOG4CXX_AUTO_TRACE(logger_);
   POLICY_LIB_CHECK(false);
 
-  ApplicationSharedPtr app;
-  uint32_t app_id = 0;
-  if (last_used_app_ids_.empty()) {
-    LOG4CXX_WARN(logger_, "last_used_app_ids_ is empty");
-    return false;
-  } else {
-    app_id = last_used_app_ids_.back();
-
-    app = application_manager_.application(app_id);
-  }
+  const uint32_t app_id = GetAppIdForSending();
+  ApplicationSharedPtr app = application_manager_.application(app_id);
```

Comment:
@jacobkeeler, yes, this issue appeared after merging External UCS functionality so in this pull request was reintroduced fix for it. I think these changes are necessary for all policy modes because of unimplemented `last_used_app_ids_`